server/src/api/bill_exec.py

```diff
@@ -53,12 +53,35 @@ def create_bill(self):
         
         try:
             username = cherrypy.session["username"]
+            #Bill has a title
+            if ((len(title) < 1)):
+                raise ValueError("no title","title")
+            #Bill has a payer
+            if ((len(payer) < 1)):
+                raise ValueError("no payer","payer")
+            #Bill has a total greater than 0
+            if (float(total)<=0):
+                raise ValueError("bad total","total")
+            #Bill has someone who needs to pay their share
+            if (len(payment_objects) < 1):
+                raise ValueError("bill only has one person","split")
+            
             success = self.database.add_bill(payer,username,title,total,payment_objects)
         except KeyError as e:
             error_msg = "User not logged in"
         except Exception as e:
-            print(e)
-            error_msg = "Unknown error occured"
+            #add the appropriate message to return
+            if(e.args[1]=="title"):
```

Comment:
This throws a tuple index out of range error for other errors that don't have the second argument.